src/fragments/lib/datastore/js/data-access/delete-snippet.mdx

```diff
@@ -1,23 +1,83 @@
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+const toDelete: Post = await DataStore.query<Post>(Post, '1234567');
+if (!toDelete) return;
+DataStore.delete<Post>(todelete);
```

Comment:
Here, and in a few other places, I might prefer:

```typescript
if (toDelete) {
  await DataStore.delete(toDelete, (post) => post.status.eq(PostStatus.INACTIVE));
}
```

It's just feels a little presumptuous to assume this is occurring in the middle of a function I want an early-exit from!